Created ExceptionsZc covergroup

coreyqh opened the pull request:
Missing c.lwsp and similar instructions

davidharrishmc commented:
Some of these loads/stores are not in Zca.  Let’s rename the covergroup to ExceptionsZc.
Missing c.lwsp, c.ldsp, c.swsp, c.sdsp, c.fld, c.fsd, c.fldsp, c.fsdsp
RV32 is also missing c.flw, c.fsw, c.flwsp, c.fswsp

coreyqh commented:
* Added  c.lwsp, c.ldsp, c.swsp, c.sdsp, c.fld, c.fsd, c.fldsp, c.fsdsp, c.flw, c.fsw, c.flwsp, c.fswsp
* Added else case to the `ifdef XLEN64` blocks for rv32 only instructions
